bin/_gcp.sh

```diff
@@ -1,5 +1,31 @@
 set -eu
 
+install_gcloud_kubectl() {
+    dir="$1"
+
+    export CLOUDSDK_CORE_DISABLE_PROMPTS=1
+    if [ -d "$dir/bin" ]; then
+        . "$dir/path.bash.inc"
+        gcloud components update
+    else
+        rm -rf "$dir"
+        curl https://sdk.cloud.google.com | bash
+        . "$dir/path.bash.inc"
```

Comment:
IIUC, this sourcing doesn't do anything, in practice, since we run this in a subshell and have to then source explicitly. I'd probably either remove the sourcing here or run this outside of a subshell when it's called